feed.xml

```diff
@@ -2,31 +2,24 @@
 search: exclude
 layout: none
 ---
-
 <?xml version="1.0" encoding="UTF-8"?>
 <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
-<channel>
-<title>{{ site.site_title | xml_escape }}</title>
-        <description>{{ site.description | xml_escape }}</description>
-        <link>{{ site.url }}</link>
-        <atom:link href="{{ "/feed.xml" | prepend: site.url }}" rel="self" type="application/rss+xml"/>
-        <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
-        <generator>Jekyll v{{ jekyll.version }}</generator>
-        {% for page in site.pages limit:20 %}
-        <item>
-            <title>{{ page.pagename | xml_escape }}</title>
-            <description><![CDATA[{{ page.content | xml_escape }}]]></description>
-            <pubDate>{{ site.date_published | date_to_rfc822 }}</pubDate>
-            <lastmod>{{ site.date_updated | date: '%Y-%m-%d' }}</lastmod>
-            <link>{{ page.url | prepend: site.url }}</link>
-            <guid isPermaLink="true">{{ page.url | prepend: site.url }}</guid>
-            {% for tag in page.tags %}
-            <category>{{ tag | xml_escape }}</category>
-            {% endfor %}
-            {% for tag in page.tags %}
-            <category>{{ cat | xml_escape }}</category>
-            {% endfor %}
-        </item>
-        {% endfor %}
-</channel>
-</rss>
+  <channel>
+    <title>{{ site.site_title | xml_escape }}</title>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ "/feed.xml" | prepend: site.url }}" rel="self" type="application/rss+xml" />
+    <description>{{ site.description | xml_escape }}</description>
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    <generator>Jekyll v{{ jekyll.version }}</generator>
+    {% assign pages = site.pages | sort: "date_published" | reverse %}
```

Comment:
@itay1313 FYI: This line governs sorting by pub date, which needs to be in reverse.